src/composables/useTreasury.ts

```diff
@@ -19,6 +19,7 @@ export function useTreasury(spaceRef: Ref<NullableSpace>) {
     if (!chainId || !wallet) return null;
 
     return {
+      shortName: networkId,
```

Comment:
We could keep `networkId` here to be consistent with others files 
We could 
```suggestion
     networkId,
```